es/chromebook_setup/instructions.md

```diff
@@ -0,0 +1,85 @@
+You can [skip right over this section](http://tutorial.djangogirls.org/en/installation/#install-python) if you're not using a Chromebook. If you
+are, your installation experience will be a little different. You can ignore the
+rest of the installation instructions.
+
+Podés [ir directamente a esta sección](http://tutorial.djangogirls.org/en/installation/#install-python) si no utilizas una Chromebook. Si lo hacés, el proceso de instalación será algo diferente. Podés ignorar el resto de las instrucciones de instalación. 
```

Comment:
This link should be different but I'm now sure how yet. This is a note to myself :)